docs/creature_text_locale.md

```diff
@@ -8,37 +8,46 @@ This table is used to provide to localized clients with localized string for cre
-| [id](#creature_text_locale-id)               | TINYINT   | UNSIGNED   | PRI | NO   | 0       |       |         |
-| [text\_loc1](#creature_text_locale-text_loc) | text         |            |     | YES  | NULL    |       |         |
-| [text\_loc2](#creature_text_locale-text_loc) | text         |            |     | YES  | NULL    |       |         |
-| [text\_loc3](#creature_text_locale-text_loc) | text         |            |     | YES  | NULL    |       |         |
-| [text\_loc4](#creature_text_locale-text_loc) | text         |            |     | YES  | NULL    |       |         |
-| [text\_loc5](#creature_text_locale-text_loc) | text         |            |     | YES  | NULL    |       |         |
-| [text\_loc6](#creature_text_locale-text_loc) | text         |            |     | YES  | NULL    |       |         |
-| [text\_loc7](#creature_text_locale-text_loc) | text         |            |     | YES  | NULL    |       |         |
-| [text\_loc8](#creature_text_locale-text_loc) | text         |            |     | YES  | NULL    |       |         |
+| Field           | Type         | Attributes | Key      | Null     | Default | Extra | Comment |
+|-----------------|--------------|------------|----------|----------|---------|-------|---------|
+| [CreatureID][1] | mediumint(8) | unsigned   |  PRI     |  NO      | 0       |       |         |
+| [GroupID][2]    | tinyint(3)   | unsigned   |  PRI     |  NO      | 0       |       |         |
+| [ID][3]         | tinyint(3)   | unsigned   |  PRI     |  NO      | 0       |       |         |
+| [Locale][4]     | varchar(4)   |            |  PRI     |  NO      |         |       |         |
+| [Text][5]       | text         |            |          |  YES     |         |       |         |
+
+[1]: #CreatureID
+[2]: #GroupID
+[3]: #ID
+[4]: #Locale
+[5]: #Text
 
 **Description of the fields**
 
-### entry
-
-This entry must be the same as [creature\_text.entry](https://trinitycore.atlassian.net/wiki/display/tc/creature_text#creature_text-entry) and then the row will be used to provide localization support for this creature record.
-
-### groupid
-
-This entry must be the same as [creature\_text.groupid](https://trinitycore.atlassian.net/wiki/display/tc/creature_text#creature_text-groupid) and then the row will be used to provide localization support for this creature record.
-
-### id
-
-This entry must be the same as [creature\_text.id](https://trinitycore.atlassian.net/wiki/display/tc/creature_text#creature_text-id) and then the row will be used to provide localization support for this creature record.
-
-### text\_loc
-
-Translated content for [creature\_text.text](https://trinitycore.atlassian.net/wiki/display/tc/creature_text#creature_text-text) field for language X.
-See [localization languages](Localization+lang) list to know which value to use for X.
-
- 
+### CreatureID
+Creature ID, obtained from [creature_template#entry](https://www.azerothcore.org/wiki/creature_template#entry)
```

Comment:
dont link the full address. correct way to do it is
```
 [creature_template#entry](creature_template#entry)
```

Reply:
> dont link the full address. correct way to do it is
> 
> ```
>  [creature_template.entry](creature_template#entry)
> ```

Ok, I will keep this in mind for other occasions.